### ISSUE TITLE
NGINX info outdated on https://nextcloud.com/collaboraonline 

### ISSUE THREAD

**Issue by tobiasKaminsky** [nextcloud/nextcloud.com#1110]
From https://github.com/nextcloud/android/issues/3650#issuecomment-491831200

NINGX info seems to be outdated.
As this is an official place we should add it directly there, same like Apache.

**Comment by jospoortvliet** [comment]
I'm all for it but where else to link to, I don't have newer info.

**Comment by tobiasKaminsky** [comment]
@juliushaertl do I remember correctly that you use ningx?

**Comment by juliushaertl** [comment]
@tobiasKaminsky Yes, but just in my dev environment. I also use a much simpler config that just proxies all locations:

```
	location / {
		proxy_pass https://collabora.host;
	}
```

The instructions from the blog post seem a bit more production ready (timeouts/headers set), they just miss proxying the capabilities endpoint.

Should we just move the apache/nginx instructions to https://github.com/nextcloud/richdocuments/tree/master/docs ?

**Comment by tobiasKaminsky** [comment]
> Should we just move the apache/nginx instructions to https://github.com/nextcloud/richdocuments/tree/master/docs ?

Is also fine by me, as it is then closer to the code, so we might not forget to update it.

**Comment by juliushaertl** [comment]
See https://github.com/nextcloud/richdocuments/pull/813

Once the docs are deployed to readthedocs automatically we can make sure that the website just references that.

**Comment by juliushaertl** [comment]
@jospoortvliet We have a documentation based on the richdocuments repo now available at https://collabora-online-for-nextcloud.readthedocs.io/en/latest/install/ with latest configuration requirements and links to upstream pages from Collabora where possible. Maybe we can replace the guide on https://nextcloud.com/collaboraonline/ with that (especially the highly outdated nginx blog link) :wink:
